web/pdf_viewer.css

```diff
@@ -141,6 +141,10 @@
   display: none;
 }
 
+.pdfViewer .page .textLayer[hidden] {
+  display: none;
+}
+
```

Comment:
All other layers simply do `div.hidden = true` in their respective `web/` JS-files, so please use the same pattern for this case well for overall consistency.

Reply:
It's a "copy/paste" from:
https://github.com/mozilla/pdf.js/blob/748be3f70215817ef5e4eeeec412556fef31908c/web/pdf_viewer.css#L140-L142

Reply:
Sure, but the textLayer should be more similar to the annotationLayer/xfaLayer/annotationEditorLayer all of which uses another format; hence my comment :-)